src/sphinx/extensions/gradle_plugin.rst

```diff
@@ -0,0 +1,358 @@
+.. _gradle-plugin:
+
+#############
+Gradle plugin
+#############
+
+This Gradle plugin integrates Gatling with Gradle, allowing to use Gatling as a testing framework.
+
```

Comment:
Missing mention of the original contributors.

Reply:
Done